taxonomies/categories.txt

```diff
@@ -25102,10 +25355,14 @@ wikidata:en:Q2797509
 
 <en:French blue-veined cheeses
 <en:Sheep Cheeses
-fr:Roqueforts
+en:Roquefort cheese from ewe's milk
```

Comment:
For cheeses, I think we should make those changes:

1. cheese -> cheeses
2. Make "Roquefort cheeses" the main name:

en:Roquerfort cheeses, Roquerfort cheeses from ewe's milk

(same in all cheeses changes below). 

The main name needs to be the most simple and obvious one.

Reply:
Thanks, fixed on next commit